bsip-0084.md

```diff
@@ -0,0 +1,213 @@
+    BSIP: 0084
+    Title: Elections based on non-core asset
+    Authors: Peter Conrad
+    Status: Draft
+    Type: Protocol
+    Created: 2019-10-12
+    Discussion: https://github.com/bitshares/bsips/issues/81
+
+# Abstract
+
+From the beginning, the BitShares blockchain has offered the ability to vote with the core token, and to have elected accounts govern the blockchain.
+
+For specific use-cases it can be desirable to have a similar mechanism for voting and elections where the votes are counted in relation not to BTS but to some other asset. An example might be a community of people who are interested in a specific topic.
+
+This BSIP proposes changes that will enable elections based on dedicated assets.
+
+
+# Motivation
+
+The feature has been requested from independent businesses as well as from within the community. In addition, it paves the way for a to-be-proposed change to BitAsset governance, i. e. the decoupling of BitAsset management from blockchain governance.
+
+
+# Rationale
+
+There are fundamental differences between the mechanics proposed here and the mechanics already in place for BTS-based voting.
+
+BTS balances are affected by almost every operation, due to transaction fees. The voting tokens will only be affected when they are being used.
+
+BTS is used in a multitude of ways, e. g. as collateral, as the counterpart in most active markets, as payment for workers and witnesses, as cashback for fees and so on. Contrarily, it is assumed that the primary purpose of the voting tokens will be voting. They are unlikely to be used as collateral.
+
+These differences allow for various simplifications and optimizations. In particular, we propose to allow only liquid balances for voting. Because these presumably change rarely in comparison to the number of distinct balances, it is more efficient to recalculate votes on the fly instead of once per maintenance interval (see STEEM for comparison).
```

Comment:
On the other hand, the simplification that only counts liquid balances does limit the potential of the feature. There could be use cases that require tokens in other forms to be counted in voting as well. I do agree to implement the simplified version first though.